mtools/mlaunch/mlaunch.py

```diff
@@ -1657,14 +1657,21 @@ def _initiate_replset(self, port, name, maxwait=30):
             print("replica set '%s' initialized." % name)
 
     def _add_user(self, port, name, password, database, roles):
-        con = self.client('localhost:%i' % port)
+        con = MongoConnection('localhost:%i'%port)
```

Comment:
I think this needs to use self.client in order to support SSL.

Reply:
Done - I think this was inadvertent.